packages/qdrant-loader/src/qdrant_loader/utils/logging.py

```
@@ -89,6 +89,45 @@ def filter(self, record):
+                    redacted = self._SENSITIVE_QUERY_PARAM.sub(
+                        r"\1***REDACTED***", path
+                    )
+                    if redacted != path:
+                        record.args = (
+                            record.args[0],
+                            record.args[1],
+                            redacted,
+                            *record.args[3:],
+                        )
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)